content/body/form-error-checking.php

```diff
@@ -138,22 +134,13 @@
       ]); ?>
 
       <p>
-        You can do the custom validation as well, but you have to ensure that
-        when the form submits and there is an error, the first input value with
-        an error receives focus so that keyboard and/or screen reader users can
-        correct mistakes easily.  You also have to ensure that form errors are marked
-        up as <code>&lt;label&gt;</code> tags for the form fields they are associated with.
+        You can do the custom validation as well, but you have to ensure that when the form is submitted and there is an error, the first input value with an error receives focus so that keyboard and/or screen reader users can correct mistakes easily. 
+        You also have to ensure that form errors are marked up as <code>&lt;label&gt;</code> tags for the form fields they are associated with.
       </p>
 
       <p>
-        The following example used
-        <a href="https://jqueryvalidation.org">jQuery.validate()</a> which is
-        not accessible. We do not recommend to use this library for new projects
-        ... it is just used as an example of how we can take existing code and
-        make it accessible. If you want information about how to make forms
-        accessible with JavaScript in the general sense, please read
-        <a           href="https://medium.com/@lsnrae/accessible-form-validation-9fa637ddb0fc"
-          >Alison Walden's excellent article on form validation</a>.
+        The following example used <a href="https://jqueryvalidation.org">jQuery.validate()</a> which is not accessible. We do not recommend using this library for new projects. 
+        it is just used as an example of how we can take existing code and make it accessible. If you want information about how to make forms accessible with JavaScript in the general sense, please read <a href="https://medium.com/@lsnrae/accessible-form-validation-9fa637ddb0fc">Alison Walden's excellent article on form validation</a>.  
```

Comment:
Please capitalize the start of the sentence: 
```suggestion
        It is just used as an example of how we can take existing code and make it accessible. If you want information about how to make forms accessible with JavaScript in the general sense, please read <a href="https://medium.com/@lsnrae/accessible-form-validation-9fa637ddb0fc">Alison Walden's excellent article on form validation</a>.  
```